tests/manage/mcg/test_host_node_failure.py

```diff
@@ -0,0 +1,202 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.framework.testlib import (
+    bugzilla,
+    ignore_leftovers,
+    ManageTest,
+    skipif_ocs_version,
+    skipif_vsphere_ipi,
+    tier4,
+    tier4a,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity, SanityExternalCluster
+from ocs_ci.helpers.helpers import (
+    wait_for_resource_state,
+    storagecluster_independent_check,
+)
+from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import (
+    get_pod_node,
+    get_noobaa_pods,
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.utility.utils import TimeoutSampler
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+@bugzilla("1853638")
+@ignore_leftovers
+@skipif_vsphere_ipi
```

Comment:
why do we skip on vsphere IPI?

Reply:
Test involving node related operations currently fails on vsphere IPI. 
This marker was added in many tests for fixing the issue #4012